docs/visual-basic/programming-guide/concepts/linq/style-part-of-a-wordprocessingml-document.md

```diff
@@ -12,7 +12,7 @@ This topic shows an example of the style part of the Office Open XML Wordprocess
  The default paragraph style has an element with the following opening tag:  
   
 ```xml  
-<w:style w:type="paragraph" w:default="1" w:styleId="Normal">  
+<w:style w:type="paragraph" w:default="1" w:styleId="Normal" />  
```

Comment:
The intro sentence states that it's an opening tag, so I wouldn't add the closing / here.